nerium/query.py

```diff
@@ -49,23 +45,30 @@ def extract_metadata(query_string):
     return metadata
 
 
-def parse_query_file(query_name):
-    """Parse query file and add query statement and metadata."""
-    query_obj = init_query(query_name)
-    try:
-        with open(query_obj.path) as f:
+def read_query_file(path):
+    if path.startswith("s3://"):
+        fs = s3fs.S3FileSystem(anon=False)
+        with fs.open(path, "rt") as f:
+            statement = f.read()
```

Comment:
While these lines are not covered (yet) by unit tests, I'm guessing this was maybe tested manually with an S3 query file?

Reply:
A bunch, yeah. I didn't add unit tests for the S3 stuff because I am a bear of very little brain, and mocks confuse me -- but I hope to noodle it out get this covered soon. But I've run it with queries on S3 a bunch at this point